internal/common/metrics/constants.go

```diff
@@ -50,12 +50,14 @@ const (
 	ActivitySucceedEndToEndLatency        = TemporalMetricsPrefix + "activity_succeed_endtoend_latency"
 	ActivityTaskErrorCounter              = TemporalMetricsPrefix + "activity_task_error"
 
-	LocalActivityTotalCounter           = TemporalMetricsPrefix + "local_activity_total"
-	LocalActivityCanceledCounter        = TemporalMetricsPrefix + "local_activity_canceled"
-	LocalActivityFailedCounter          = TemporalMetricsPrefix + "local_activity_failed"
-	LocalActivityErrorCounter           = TemporalMetricsPrefix + "local_activity_error"
-	LocalActivityExecutionLatency       = TemporalMetricsPrefix + "local_activity_execution_latency"
-	LocalActivitySucceedEndToEndLatency = TemporalMetricsPrefix + "local_activity_succeed_endtoend_latency"
+	LocalActivityTotalCounter             = TemporalMetricsPrefix + "local_activity_total"
+	LocalActivityCanceledCounter          = TemporalMetricsPrefix + "local_activity_canceled" // Deprecated: Use LocalActivityExecutionCanceledCounter instead.
+	LocalActivityExecutionCanceledCounter = TemporalMetricsPrefix + "local_activity_execution_cancelled"
+	LocalActivityFailedCounter            = TemporalMetricsPrefix + "local_activity_failed" // Deprecated: Use LocalActivityExecutionFailedCounter instead.
+	LocalActivityExecutionFailedCounter   = TemporalMetricsPrefix + "local_activity_execution_failed"
+	LocalActivityErrorCounter             = TemporalMetricsPrefix + "local_activity_error"
```

Comment:
What's the difference between these 2?
I don't see `local_activity_error` in our docs.

Reply:
I didn't touch `local_activity_error ` in this PR. Looks like it counts how many times local activities panic

Reply:
And what does `local_activity_execution_failed` track?

Reply:
`local_activity_execution_failed ` tracks if the local activity fails. If the activity panics both are incremented